source/globalCommands.py

```diff
@@ -898,19 +898,23 @@ def script_navigatorObject_currentDimensions(self,gesture):
 	script_navigatorObject_currentDimensions.__doc__=_("Reports information about the location of the text or object at the review cursor. Pressing twice may provide further detail.") 
 	script_navigatorObject_currentDimensions.category=SCRCAT_OBJECTNAVIGATION
 
+	@script(
+		description=_(
+			# Translators: Input help mode message for move navigator object to current focus command.
+			"Sets the navigator object to the current focus,"
+			"and the review cursor to the position of the caret inside it, if possible."
+		),
+		category=SCRCAT_OBJECTNAVIGATION,
+		gestures=("kb:NVDA+numpadMinus", "kb(laptop):NVDA+backspace"),
+	)
 	def script_navigatorObject_toFocus(self,gesture):
-		obj=api.getFocusObject()
-		try:
-			pos=obj.makeTextInfo(textInfos.POSITION_CARET)
-		except (NotImplementedError,RuntimeError):
-			pos=obj.makeTextInfo(textInfos.POSITION_FIRST)
-		api.setReviewPosition(pos)
+		TIAtCaret = self._getTIAtCaret(True)

```

Comment:
please start variables with lower case, otherwise they look like classes / types
```suggestion
		tIAtCaret = self._getTIAtCaret(True)

```

Reply:
Done so. I've decided to start the variable name with upper case T just because PEP 8 recommends to capitalize all letters of the acronyms and ti is obviously short for TextInfo.